src/atoms/TagLink/tests/TagLink.test.js

```diff
@@ -5,28 +5,26 @@ import TagLink from '../TagLink';
 // automatically unmount and cleanup DOM after the test is finished.
 afterEach(cleanup);
 
-describe('Taglink tests', () => {
-  test('TagLink props value matches as expected', () => {
-    // Arranged
-    const tag = {
-      href: '/the/url/path',
-      tagName: 'Live from Here'
-    };
-    // Act
-    const { getByText } = render(
-      <TagLink tagName={tag.tagName} href={tag.href} />
-    );
-    const TagLinkNode = getByText('Live from Here');
-    // Assert
-    expect(TagLinkNode.getAttribute('href')).toBe('/the/url/path');
-    expect(TagLinkNode.innerHTML).toBe('Live from Here');
-  });
-  test('Throws an error when a isRequired value is missing', () => {
-    expect(() => {
-      render(<TagLink />);
-    }).toThrow();
-  });
-  test("We'll have to overhaul TagLink later anyway", () => {
-    expect(true).toBe(true);
-  });
+test('TagLink props value matches as expected', () => {
```

Comment:
This is kind of a bad name that's held over from the original test. It's hard to tell what's being tested here from that. In this case, I'd probably say that renaming this to "can set tagName and href via props" would be a lot clearer.